x-pack/plugins/endpoint/public/embeddables/resolver/view/defs.tsx

```diff
@@ -5,7 +5,7 @@
  */
 
 import React, { memo } from 'react';
-import { saturate } from 'polished';
+import { saturate, lighten } from 'polished';
```

Comment:
🔴 I think these assets were swapped out for new ones from Creative a while back. I think you can safely jettison the changes from this file. They really aren't related to the rest of this PR.

Reply:
🎗️ remove from defs on upstream merge